Fixing issues #48-50 and #45

HenryPajuri opened the pull request:
Change password hetkel veel vigane, login /logout nupp ei ole hetkel navbari peal

HenryPajuri committed:
Main page alustatud, nav-bar, login ühenduses supabasega

HenryPajuri committed:
Main page, my profile page, login

HenryPajuri committed:
Fixing issues #48-50 and #45 clearing files

Some issues still need some work

Kassukas123 commented:
Sul hetkel on conflictid main branchiga, kuna ennem oli minu varajane commit õhus, kuid nüüd on see mainiga mergitud:

Conflicting files
app/globals.css
app/layout.tsx
app/login/page.tsx
app/majandus/page.tsx
app/my_profile/page.tsx
app/page.tsx
app/sport/page.tsx
package-lock.json

Ma laseks sul need conflictid ennem ära parandada ja siis uus commit teha. Teine variant on see, et pulli see uus "main" branch endale ja pane oma uus kood sinna juurde ja commiti see. Sedasi saab kindel olla, et kõik conflictid on lahendatud.

Exortile commented:
Lisan igaks juhuks juurde, `package-lock.json` faili juures suht suva kuidas conflicti fixid, aga kõik muud failid on olulised. Ja teiseks, see kommentaar on ka veel reminder conflictid ära fixida :D

HenryPajuri committed:
Changepassword works, ChangeEmail works, login/logout at every page

Added some buttons and functionalities to my_profile page. Also login and logout button is now at every page. Mage my_profile look a little better. Yet to get delete user, firstname/lastname adding working.

HenryPajuri committed:
Merge branch 'main' into main
